_processSteps.py

```diff
@@ -348,75 +324,117 @@ def step_template(inps):
 
     return inps
 
-###############################################################################
 
+###############################################################################
 
-def step_ssara(inps):
-    if inps.flag_ssara:
-        if not os.path.isdir(inps.slcDir):
-            os.mkdir(inps.slcDir)
-        if inps.slcDir is not inps.work_dir + '/SLC' and not os.path.isdir(inps.work_dir + '/SLC'):
-            os.symlink(inps.slcDir, inps.work_dir + '/SLC')
 
-        putils.call_ssara(inps.custom_template_file, inps.slcDir)
+def call_ssara(flag_ssara, custom_template_file, slc_dir):
+    """ Downloads data with ssara and asfserial scripts. """
 
-    if inps.stopssara:
-        logger.debug('Exit as planned after ssara')
-        sys.exit(0)
+    if flag_ssara:
+        out_file = os.getcwd() + '/' + 'out_download_ssara'
+        command = 'download_ssara_rsmas.py ' + custom_template_file
+        messageRsmas.log(command)
+        command = '('+command+' > '+out_file+'.o) >& '+out_file+'.e'
+        command_ssh = 'ssh pegasus.ccs.miami.edu \"s.cgood;cd ' + slc_dir + '; ' +  command + '\"'
+        status = subprocess.Popen(command_ssh, shell=True).wait()
+        print('Exit status from download_ssara_rsmas.py:',status)
 
-    return inps
+        out_file = os.getcwd() + '/' + 'out_download_asfserial'
+        command = 'download_asfserial_rsmas.py ' + custom_template_file
+        messageRsmas.log(command)
+        command = '('+command+' > '+out_file+'.o) >& '+out_file+'.e'
+        command_ssh = 'ssh pegasus.ccs.miami.edu \"s.cgood;cd ' + slc_dir + '; ' +  command + '\"'
+        status = subprocess.Popen(command_ssh, shell=True).wait()
+        print('Exit status from download_asfserial_rsmas.py:',status)
+    
+    return None
 
 ###############################################################################
-def step_runfiles(inps):
```

Comment:
step_runfiles and the functions inside are removed and replaced with two separate steps of:
create_or_copy_dem --> which calls dem_rsmas.py
and create_runfiles --> which calls create_stacksentinel_run_files.py